### DISCUSSION TITLE
Vercel build `An unexpected internal error occurred`

### DISCUSSION THREAD

**Discussion by lukebussey** [vercel/community#570] [Help] [answered]
I'm receiving an error on a NextJS 12 site that has always previously deployed fine:

```
Traced Next.js server files in: 292.843ms
--
21:21:36.718 | Created all serverless functions in: 3.483s
21:21:38.920 | Uploading build outputs...
21:21:42.485 | An unexpected internal error occurred
```

All branches, even previously working ones now receive the same error so I'm stuck. Is there any way to find out what's happening?

**Comment by meerface** [comment]
Yes happening to us too, and a few more people here: https://github.com/vercel/community/discussions/436

**Comment by swarnava** [comment]
@lukebussey @meerface We are currently investigating this and update you shortly via Support ticket. Apologize for the inconvenience.

**Comment by swarnava** [comment, marked as answer]
Can you redeploy and let us know if you are still facing issue?

**Reply by lukebussey** [reply]
Working fine now. Thanks!
